roles/java/meta/argument_specs.yml

```diff
@@ -0,0 +1,88 @@
+---
+argument_specs:
+  main:
+    short_description: Main entrypoint for the java role
+    options:
+      java_keystore:
+        type: dict
+        options:
+          path:
+            description: |
+              Local path to the java keystore
+            required: true
+            type: str
+          pass:
+            description: |
+              Password to unlock the java keystore
+            required: true
+            type: str
```

Comment:
not sure we could have this as a role argument, how it fits with the pre-existing ansible-vault feat?

Reply:
What I had in mind was:
* a we add a new secret in the `Populate secrets.yml with missing secrets` task (e.g.  `keystore_password`)
* pass that var as a role argument when calling the java role (`repository` molecule converge example below).

```yaml
- name: Converge
  hosts: all
  roles:
    - role: postgres
    - role: java
      java_keystore:
        type: JKS
        path: "{{ content_folder }}/keystore/alfresco.jks"
        pass: "{{ keystore_password }}"
        cert_containers:
          - path: "{{ inventory_hostname }}.p12"
            pass: "{{ keystore_password }}"
            trustcachain: true
    - role: activemq
    - role: repository
```